src/app/pages/blog/blog-pages.tsx

```diff
@@ -0,0 +1,106 @@
+import React, {useEffect} from 'react';
```

Comment:
These routines were extracted from blog.js (the next file down). Types were added. If you had the two files in separate windows, it would be easy to see.